docs/features/details/Details.md

```diff
@@ -29,47 +27,58 @@ An object's further detail properties as entries with
                  e.g. a simple string or a more complex object
 end note
 
+Branch --> Details : details
+Build --> Details : details
+UseCase --> Details : details
+Scenario --> Details : details
+Step --> Details : details
+
+class Object
+
+Details --> "*" Object
+
+
 class String
 
+Object <|-- String
+
+
 class ObjectDescription {
     name: String
     type: String
     details: Details
 }
 
+Object <|-- ObjectDescription
+
+
 class ObjectReference {
     name: String
     type: String
 }
 
+Object <|-- ObjectReference
+
+
 class ObjectList {
     items: List<Object>
 }
 
+Object <|-- ObjectList
+ObjectList --> "*" Object : items
+
+
 class ObjectTreeNode {
     item: Object
     details: Details
     children: List<ObjectTreeNode>
 }
 
-Branch --> Details : details
-Build --> Details : details
-UseCase --> Details : details
-Scenario --> Details : details
-Step --> Details : details
-
-Details --> "*" Object
-
-Object <|-- String
-Object <|-- ObjectDescription
-Object <|-- ObjectReference
-Object <|-- ObjectList
 Object <|-- ObjectTreeNode
-
-ObjectList --> "*" Object : items
 ObjectTreeNode --> "1" Object : item
 ObjectTreeNode --> "*" ObjectTreeNode : children
 
```

Comment:
The association from _ObjectTreeNode_ to _Details_ is missing.
Or is it omitted on purpose so the diagram isn't too cluttered?

Reply:
as discussed - yes not to clutter

Reply:
we decided to ommit on purpose